suggestions/service.py

```diff
@@ -0,0 +1,87 @@
+import abc
+import logging
+
+from mailjet_rest import Client
+
+from .model import Suggestion
+
+
+class SuggestionService(abc.ABC):
+    """
+    Service to send a suggestion
+    """
+
+    @abc.abstractmethod
+    def add_suggestion(self, suggestion: Suggestion):
+        """
+        Method to send a suggestion
+        """
+
+
+class MailjetSuggestionService(SuggestionService):
+    def __init__(
+        self,
+        mailjet_client: Client,
+        suggestion_email_sender: str,
+        suggestion_email_recipient: str,
+        suggestion_name: str,
+        suggestion_mailjet_custom_id: str,
+    ):
+        self.mailjet_client = mailjet_client
+        self.suggestion_email_sender = suggestion_email_sender
+        self.suggestion_email_recipient = suggestion_email_recipient
+        self.suggestion_name = suggestion_name
+        self.suggestion_mailjet_custom_id = suggestion_mailjet_custom_id
+        self.logger = logging.getLogger(__name__)
+
+    def add_suggestion(self, suggestion: Suggestion):
+        data = {
+            "Messages": [
+                {
+                    "From": {
+                        "Email": self.suggestion_email_sender,
+                        "Name": suggestion.name,
+                    },
+                    "To": [
+                        {
+                            "Email": self.suggestion_email_recipient,
+                            "Name": self.suggestion_name,
+                        }
+                    ],
+                    "Subject": suggestion.title,
```

Comment:
Just checked the requirements for this task and noticed that the `title` is not a field we should receive. I think we could remove this field and add a default value here like:

`f"Querido Diário, hoje recebi uma sugestão de {suggestion.name}"`